### DISCUSSION TITLE
How i can compare stored date attribute month and day with todays date

### DISCUSSION THREAD

**Discussion by bladerunner2912** [Automattic/mongoose#12963] [Q&A]
I want to run a query from my Node.js typescript server finding users  whose birthday is today , in next and last 5 days. I cant use $where in my query. Whats the best approach I can do

```
` presentEvent = await User.find({
      dob: {
        "$month": today.getMonth(),
      }
    `});``
```

**Comment by vkarpov15** [comment]
`dob: { $gte: addDays(today, -5), $lte: addDays(today, 5) }`, using [date-fns addDays()](https://snyk.io/advisor/npm-package/date-fns/functions/date-fns.addDays)
